projects/tests.py

```diff
@@ -97,240 +99,150 @@ def setUpTestData(cls):
-
-    def test_acronym_max_length(self):
-        project = self.test_project
-        max_length = project._meta.get_field("acronym").max_length
-        self.assertEqual(max_length, 20)
-
-    def test_impact_level_max_length(self):
-        project = self.test_project
-        max_length = project._meta.get_field("impact_level").max_length
-        self.assertEqual(max_length, 20)
-
-    def test_location_max_length(self):
-        project = self.test_project
-        max_length = project._meta.get_field("location").max_length
-        self.assertEqual(max_length, 100)
-
-    def test_status_max_length(self):
-        project = self.test_project
-        max_length = project._meta.get_field("status").max_length
-        self.assertEqual(max_length, 20)
-
-    # Tests for default value
-    def test_status_default_value(self):
-        # create a project without specifying status
-        Project.objects.create(
-            title="Basic Project",
-            acronym="BP",
-            catalog_version="NIST 800-53",
-            impact_level="low",
-            location="other",
-            creator=self.test_user,
-            catalog=self.test_catalog,
+            creator=user,
+            catalog=test_catalog,
         )
 
-        # ensure project status defaults as expected
-        expected_status = "active"
-        project = Project.objects.get(title="Basic Project")
-        status = project.status
-        self.assertEqual(status, expected_status)
-
-        Project.objects.create(
-            title="Test Project",
-            acronym="TP",
-            catalog_version="NIST 800-53",
-            impact_level="low",
-            location="other",
-            creator=User.objects.get(id=1),
-            catalog=self.test_catalog,
+    def test_project_permissions(self):
+        self.assertTrue(
+            self.user.has_perms(("change_project", "view_project", "manage_project_users"), self.test_project)
         )
-        project = Project.objects.get(title="Test Project")
-        user = User.objects.get(id=1)
-        self.assertEqual("change_project" in get_perms(user, project), True)
-        self.assertEqual("view_project" in get_perms(user, project), True)
-        self.assertEqual("manage_project_users" in get_perms(user, project), True)
-
-    def test_default_component(self):
-        project = self.test_project
-        has_default = False
-        for c in project.components.all():
-            if c.title == "Pretty Ordinary Project private" and c.status == 1:
-                has_default = True
 
-        self.assertTrue(has_default)
+    def test_project_has_default_component(self):
+        private_component = self.test_project.components.get(title="Pretty Ordinary Project private")
+        self.assertEqual(private_component.status, Component.Status.SYSTEM)
 
 
-class ProjectRequiredFieldsTest(AuthenticatedAPITestCase):
+class ProjectCreateViewTestCase(AuthenticatedAPITestCase):
```

Comment:
I noticed that we didn't have a test that actually tried to create a project with valid data, so I refactored/extended this test case to include a test for attempting to create a "legit" project while consolidating the "required fields" tests into a single, parametrized test.